puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/packaging_build_rpm.groovy

```diff
@@ -0,0 +1,77 @@
+pipeline {
+    agent { label 'rpmbuild' }
+
+    options {
+        timestamps()
+        timeout(time: 2, unit: 'HOURS')
+        disableConcurrentBuilds()
+        ansiColor('xterm')
+    }
+
+    stages {
+        stage("Setup Environment") {
+            steps {
+                git(url: 'https://github.com/theforeman/foreman-packaging/', branch: env.branch)
+                setup_obal()
+            }
+        }
+
+        stage("Apply Pull Request Changes") {
```

Comment:
Up until 1.17 we used the packaging_build_rpm job for foreman-packaging PRs as well but we no longer do that. I think that nothing calls this task with a PR anymore and you can remove this.